docs/services/slam/cartographer/_index.md

```diff
@@ -457,15 +305,14 @@ In the meantime, your machine will show up at the map's origin (with the `(x,y)`
 
 ### Localize only
 
-In this mode, the `cartographer` module on your machine executes the Cartographer algorithm itself locally to find its position on a map.
+In this mode, the `cartographer` module on your machine executes the Cartographer algorithm to find its position on a map. This mode is better when using motion planning because the map will no longer be modified.
```

Comment:
```suggestion
In this mode, the `cartographer` module on your machine executes the Cartographer algorithm to find its position on a map. This mode is better when using motion planning because the map will not be modified.
```